app/packs/stylesheets/dashboard.scss

```diff
@@ -38,4 +38,5 @@
 
 ul.collections {
   list-style-type: none;
+  padding-left: 0rem;
```

Comment:
I believe the convention is to remove the units for a "0" value. See https://github.com/CSSLint/csslint/wiki/Disallow-units-for-zero-values

Reply:
Ahhh ... fixed.